analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    geod = Geod(ellps= 'WGS84') 
</pre><p>nit: when setting named arguments, we don't put a space after the "="</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>